lnd_methods/onchain/open_channel.js

```diff
@@ -117,6 +123,8 @@ module.exports = (args, cbk) => {
           spend_unconfirmed: !minConfs,
           use_base_fee: args.base_fee_mtokens !== undefined,
           use_fee_rate: args.fee_rate !== undefined,
+          zero_conf: !!args.is_trusted_funding,
+          commitment_type: !!args.is_trusted_funding? 3 :undefined, // Anchors
```

Comment:
```suggestion
          commitment_type: !!args.is_trusted_funding? 'ANCHORS' : undefined, // Anchors
```

I think it should be set to the enum value itself, that's how openChannels does it